### ISSUE TITLE
learning rate decay?

### ISSUE THREAD

**Issue by noeagles** [pytorch/pytorch#4064]
can we have learning rate decay?

**Comment by lanpa** [comment]
http://pytorch.org/docs/0.3.0/optim.html?highlight=lr_scheduler#how-to-adjust-learning-rate
